### PR TITLE
 New A000566 Jp Hayek

### PR DESCRIPTION
**Pull request by jphayek**
C'est la nouvelle version.

### CODE REVIEW

File: oeis.py [hunk 1]
```diff
@@ -9,8 +9,7 @@
 import sys
 import os
 
-import numpy as np
-import matplotlib.pyplot as plt
```
Comment (JulienPalard) [inline oeis.py]: This will break some other functions, please don't remove those lines.

File: oeis.py [hunk 2]
```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
...
+
+        x, y = 1, 1
+
+        yield 0
+
+        while True:
+
+            yield x
+
+            x, y = x + y + 5, y + 5
```
Comment (JulienPalard) [inline oeis.py]: Please avoid writing on every other line, compact this, and run black just to add mandatory blank lines.

File: oeis.py [hunk 3]
```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
...
+
+    A000566 = aList()
+    return [next(A000566) for i in range(50)]
```
Comment (JulienPalard) [inline oeis.py]: You hardcoded 50 here, dont hardcode the limit, there's the `limit` parameter for this.

File: oeis.py [hunk 4]
```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(sequence[i - 1] + sequence[i - 2])
     return sequence
 
+@oeis
+def A000566(start: int = 0, limit: int = 20) -> Collection[int]:
+    def aList():
```
Comment (JulienPalard) [inline oeis.py]: I don't like this function name, also PEP8 tells it should use snake_case not camelCase.

Maybe:
```suggestion
    def A000566_generator():
```

File: oeis.py [hunk 5]
```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(sequence[i - 1] + sequence[i - 2])
     return sequence
 
+@oeis
+def A000566(start: int = 0, limit: int = 20) -> Collection[int]:
```
Comment (JulienPalard) [inline oeis.py]: Your function accepts a `start` value, you should use it.

Comment (jphayek) [reply oeis.py]: Ah oui !

File: oeis.py (hunk 4 above)
Comment (jphayek) [reply oeis.py]: Oui vous aviez raison ... Désolé

File: oeis.py (hunk 3 above)
Comment (jphayek) [reply oeis.py]: D'accord

File: oeis.py [hunk 6]
```diff
@@ -8,11 +8,12 @@
 from typing import Collection, Dict, List, Callable
 import sys
 import os
-
```
Comment (JulienPalard) [inline oeis.py]: This blank line was mandatory to split between Python stdlib imports and PyPI installed imports.

File: oeis.py [hunk 7]
```diff
@@ -8,11 +8,12 @@
...
 import numpy as np
 import matplotlib.pyplot as plt
 
 
+
+
```
Comment (JulienPalard) [inline oeis.py]: 4 blanks lines are a bit too much here, if you run black I bet you'll end up with one or two, which is enough.

**Comment by JulienPalard** [comment]
ping.

**Comment by JulienPalard** [comment]
This PR conflicts a lot since recent improvements and refactoring, and is inactive. Don't hesitate to re-open it if you want to rebase your work.

**Pull request closed without merging**